sonicwall/icon_sonicwall/util/util.py

```diff
@@ -1,11 +0,0 @@
-import re
-
-def determine_address_type(address: str) -> str:
```

Comment:
Did we want to remove this?

Reply:
@jschipp-r7 Yes, that code wasn't used anywhere except from within my action.